desc/objectives/linear_objectives.py

```diff
@@ -160,7 +160,10 @@ def build(self, eq, use_jit=True, verbose=1):
 
     def compute(self, *args, **kwargs):
         """Compute deviation from desired boundary."""
-        x = kwargs.get(self.args[0], args[0])
+        if len(args):
+            x = kwargs.get(self.args[0], args[0])
+        else:
+            x = kwargs.get(self.args[0])
```

Comment:
Why would there ever be more than one argument? `FixBoundaryR` and `FixBoundaryZ` should take only `R_lmn` or `Z_lmn`.

Reply:
I don't quite remember the issue that prompted this, @f0uriest do you remember? Something to do with R_lmn and Rb_lmn I think

The direct issue I think was that if the passed-in just one arg, args[0] throws an indexing error, and this if statement prevents that.

Reply:
The issue isn't more than 1 arg, its 0 args. IE, if you just pass things as kwargs by unpacking a dict. This will make sure that it works whether you pass it as a positional or keyword argument.